### ISSUE TITLE
Blue notifier button, mail.google.com is set to always alllow cookies

### ISSUE THREAD

**Issue by Schmiggy** [inbasic/ignotifier#547]
I cant figure this out for the life of me, but some time ago the right click, logged in accounts option stopped working for me and has been greyed out.  

Cookies for mail.google.com are set to always allow.  

I have all three accounts logged in to my google profile on the upper right bar, with my account1 as the default.  

I have three accounts in the extension options:

account1
account2
account3

Ive also tried with the inbox flags, I believe I am doing that right, as "account1-inbox"

basically the only working functionality I have is clicking on the notifer envelope and going to my primary account.  Any help on this?
